documentation/howto-devonfw-CobiGen.adoc

```diff
@@ -0,0 +1,463 @@
+:toc:
```

Comment:
It seems that you have created a complete new file called howto-devonfw-CobiGen.adoc. Instead, you should have modified existing document [howto-devonfw-ide-CobiGen-PoC-E2E.adoc](https://github.com/devonfw/cobigen/blob/master/documentation/howto-devonfw-ide-CobiGen-PoC-E2E.adoc)

Reply:
After a quick chat with @sujith-mn I remembered that we wanted to merge the OpenAPI and Entity generation in one document. That's why Sujith created a new one.

What is pending is the removal of the old docs. Please @sujith-mn remove them.